src/openrct2/platform/shared.c

```diff
@@ -763,7 +765,21 @@ void platform_set_cursor_position(sint32 x, sint32 y)
 
 uint32 platform_get_ticks()
 {
-	return SDL_GetTicks();
+#ifdef _WIN32
+    return GetTickCount();
+#else
+    struct timespec ts;
+    if (clock_gettime(CLOCK_MONOTONIC, &ts) != 0) {
+        log_fatal("clock_gettime failed");
+        exit(-1);
+    }
+    return (uint32)(ts.tv_nsec / 1000000);
+#endif
+}
+
+void platform_sleep(uint32 ms)
+{
+    SDL_Delay(ms);
```

Comment:
Yea ok, although I might call `Sleep` conditionally for Windows as its much cheaper than the code generated by `sleep_for`.

http://stackoverflow.com/questions/32904371/windows-7-overshoot-c-stdthis-threadsleep-for/32963363